test/apps/iodemo/io_demo.cc

```diff
@@ -1376,12 +1376,21 @@ class DemoClient : public P2pDemoCommon {
         ++server_info.num_sent[op];
         ++_num_sent;
         server_info.bytes_sent[op] += data_size;
+
         if (get_num_uncompleted(server_info) == opts().conn_window_size) {
             active_servers_remove(server_index);
         }
 
         assert(server_info.num_completed[op] < server_info.num_sent[op]);
         assert(_num_completed < _num_sent);
+
+        if (opts().min_data_size != 0) {
+            assert(server_info.bytes_completed[op] <
+                   server_info.bytes_sent[op]);
+        } else {
+            assert(server_info.bytes_completed[op] <=
+                   server_info.bytes_sent[op]);
```

Comment:
what the difference between these asserts?

Reply:
if `min_data_size` is `0`, then we sum of `server_info.bytes_sent[op]` and `data_size` could be equal to either `server_info.bytes_sent[op]` (if `data_size == 0`) or > `server_info.bytes_sent[op]` (if `data_size` > `0`). So, `server_info.bytes_completed[op]` could be equal to `server_info.bytes_sent[op]`
but if `min_data_size != 0`, then `server_info.bytes_completed[op]` must be lower than `server_info.bytes_sent[op]` in any case.